app/decorators/valkyrie/resource_decorator.rb

```diff
@@ -79,6 +79,10 @@ def orderable_files?
     manageable_files?
   end
 
+  def manageable_order?
+    true unless NumismaticMonogramDecorator
```

Comment:
Can you override this in `NumismaticMonogramDecorator` instead of having a conditonal here?

Reply:
yes I figured out I was wrong